instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -192,6 +200,10 @@ def _traced_execute_command(func, instance, args, kwargs):
             if callable(request_hook):
                 request_hook(span, instance, args, kwargs)
             response = func(*args, **kwargs)
+            if span.name == "redis.search":
```

Comment:
I think these should be within the `span.is_recording()` check.

Reply:
Changed